apis/server/cri_bridge.go

```diff
@@ -0,0 +1,30 @@
+package server
+
+import (
+	"context"
+	"net/http"
+)
+
+func (s *Server) criExec(context context.Context, rw http.ResponseWriter, req *http.Request) (err error) {
+	if s.StreamRouter == nil {
+		return EncodeResponse(rw, http.StatusNotImplemented, nil)
+	}
+	s.StreamRouter.ServeExec(rw, req)
```

Comment:
`s.StreamRouter.ServeExec(rw, req)`, Why return errors ?

Reply:
I still think a router handler should return an error, WDYT?   @zhuangqh @Starnop 

But I think we no need fix this in this PR